utils/supabase.js

```diff
@@ -5,6 +5,28 @@ const supabase = createClient(
   process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY
 );
 
+// NOTE: Changing this will create breaking changes in the Database
+export const NUM_OF_DIVES = {
```

Comment:
Added this constants to start storing `ints` in the DB instead of text. That way we can change the text to how we please. The DB should not be in charge of saving the exact text we want to display to the user (lots of reasons of why that)